content/applications/sales/members.rst

```diff
@@ -0,0 +1,91 @@
+=======
+Members
+=======
+
+The *Members* application is where all operations related to memberships can be configured and
+managed. The *Members* app integrates with the *Sales* and *Accounting* applications to sell and
+invoice memberships directly to customers.
+
+Membership products
+===================
+
+To create a new membership product, navigate to :menuselection:`Members app --> Configuration -->
+Membership Products`, and click :guilabel:`New` to open a blank product record.
+
+Complete the blank form with the necessary information, including the :guilabel:`Membership
+Duration`.
+
+.. note::
+   Membership products require a start and end date, as they are used to determine :ref:`membership
+   status <sales/membership-status>`. Membership products can be sold *before* their active start
+   date.
+
+.. image:: members/membership-product.png
+   :align: center
+   :alt: A new membership product in the members app.
+
+Membership products can be added to a sales order, and invoiced as regular products or
+subscriptions.
+
+Activate a membership
+=====================
+
+To activate a membership from the *Contacts* application, navigate to :menuselection:`Contacts app`,
+and click on a contact to open that specific contact's detail form.
+
+From the resulting contact form, open the :guilabel:`Membership` tab, and click :guilabel:`Buy
+Membership`.
+
+.. tip::
+   To successfully buy a membership, both the :guilabel:`Account Receivable` and :guilabel:`Account
+   Payable` lines must first be configured for that contact. These can be found in the contact
+   detail form in the :guilabel:`Accounting` tab under the :guilabel:`Accounting entries` heading.
```

Comment:
QQ: this section is not present in the `17.0` version of this doc, was that on purpose?

Reply:
Yes, when following the flow for 16, I found it was necessary to have these 2 fields configured before buying a membership to prevent an error.